### DISCUSSION TITLE
Inventory item return feature

### DISCUSSION THREAD

**Discussion by zeradawit** [Ayagikei/LifeUp#151] [General]
Can someone explain how the return feature works for inventory items? I often find myself unable to return any items in my inventory, regardless of how I obtained them. When I try to return an item, I don't receive the same return value that the item sells for in the shop.

**Comment by Ayagikei** [comment]
Hello,
I'm not sure what you mean by "unable to return". Please provide more detailed information. Normally, you can long-press the items in the inventory to return them in the menu bar.

The refunded value is currently only 80% of the selling price. This value can be adjusted in the settings in the upper right corner of the store.

**Comment by zeradawit** [comment]
Thank you very much. It was my mistake. I thought the "discount rate for returning" option only affected the selling price of an item when I returned to buy it again. Actually, it doesn't make any sense for it to work that way, so I set the value to zero. Now I see the mistake I made.
So the returning option works perfectly fine and in more smart way beyond my expectation.
